lib/util/news.ts

```diff
@@ -0,0 +1,43 @@
+import type { Languages } from '../hooks/useLanguage'
+import { languages } from '../hooks/useLanguage'
+import { z } from 'zod'
+
+export type News = {
+  title: string,
+  date: Date,
+  description: (string | URL)[],
```

Comment:
Just a thought but you may want to allow (a subset of) markdown here

I'd say "inline formatting" such as bold, italic, strikethrough, links, inline code, as well as lists sound like a reasonable subset

I'm not sure if there is some special nextjs ecosystem solution for this which allows such a fine-grained level of control but you could always try to use markdown-it or some other markdown parser which tend to be more configurable on that front